system/draft/test_misc.py

```diff
@@ -96,28 +90,67 @@
     print(res)
 
 
+# TODO parametrize this to read as trustee and as default user
 @pytest.mark.asyncio
```

Comment:
Please make sure that all tests are run as part of nightly builds.

Reply:
It will be done in the next PR.